scout/server/extensions/chanjo2_extension.py

```diff
@@ -15,14 +16,15 @@
 class Chanjo2Client:
     """Runs requests to chanjo2 and returns results in the expected format."""
 
-    def mt_coverage_stats(self, individuals: dict) -> Dict[str, dict]:
+    def mt_coverage_stats(self, case_obj: dict) -> Dict[str, dict]:
```

Comment:
👍 I guess it introduces a slight difference in the API vs chanjo, but we always call them separately anyway; no polymorphism here! 😁